### ISSUE TITLE
Getting periodic 486 and 503 errors

### ISSUE THREAD

**Issue by jeffwhelpley** [drachtio/drachtio-srf#158]
Periodically (i.e. 3 to 10 times per hour) we are getting SIP errors from our Drachtio listener like this:

```
Sip non-success response: 486
```

or 

```
Sip non-success response: 503
```

There are no other details in the error that gets returned, but looking up online it appears that both are related to scalability. Assuming that is the case, what adjustments should I make to our Drachtio instance to be able to support more simultaneous calls?

**Comment by davehorton** [comment]
Its not clear if you are sending INVITEs and receiving those responses in return, or if you are receiving INVITEs and generating those responses in return.  In general, a full SIP log showing the INVITE transaction would be necessary to learn more

**Comment by davehorton** [comment]
can you supply a full log @jeffwhelpley ?

**Comment by jeffwhelpley** [comment]
Yes, I will get logs and attach here later this week. I have to re-run our full production test to get the logs.
